index.html

```diff
@@ -238,9 +238,9 @@ <h3 id="selectHardwareAcceleration">Hardware Acceleration</h3>
 														<div class="col-lg-3 col-md-3 r-option version" role="option" tabindex="-1" aria-selected="false" id="CANN">
 															<span>CANN (Preview)</span></div>
 														<div class="col-lg-3 col-md-3 r-option version" role="option" tabindex="-1" aria-selected="false" id="MIGraphX">
-															<span>MIGraphX (Preview)</span></div>	
+															<span>MIGraphX </span></div>
 														<div class="col-lg-3 col-md-3 r-option version" role="option" tabindex="-1" aria-selected="false" id="ROCm">
-															<span>ROCm (Preview)</span></div>		
```

Comment:
Our criteria for removing "preview" is that it has shipped in a production scenario. Can you confirm this is the case? 
Also, if removing preview, please shift the order of these up in the matrix. The ordering used is: alphabetical, with all non-preview before preview providers

Reply:
Will do. Let me adjust this

It's my understating that this is in production as this is used and tested with onnxruntime in mind